src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -85,7 +91,7 @@ static bool SetEqualsKeys(ConcurrentDictionary<string, bool> left, ConcurrentDic
                 var keysLeft = left.Keys;
                 var keysRight = right.Keys;
 
-                return keysLeft.Count == keysRight.Count && keysLeft.SequenceEqual(keysRight, StringComparers.ResourceType);
+                return keysLeft.Count == keysRight.Count && keysLeft.OrderBy(key => key).SequenceEqual(keysRight.OrderBy(key => key), StringComparers.ResourceType);
```

Comment:
Should the OrderBy also use StringComparers.ResourceType?

Reply:
Yea, that's a good idea